src/Uno.UITest.Helpers/AppInitializer.cs

```diff
@@ -50,10 +50,10 @@ public class AppInitializer
 		/// </summary>
 		/// <remarks>This method is generally called from the type constructor of a test assembly.</remarks>
 		/// <returns>An <see cref="IApp"/> instance representing the running application.</returns>
-		public static IApp ColdStartApp()
+		public static IApp ColdStartApp(Xamarin.UITest.Configuration.AppDataMode? mode = null)
```

Comment:
I had to make it nullable and default to null since the existing logic used different modes in different code paths. I don't like it being nullable, but this seems like the only way to preserve the existing behavior?

Reply:
Why not just make another overload? That'll provide the proper default value if needed. Otherwise it'll do.